src/utils/FileOperations.java

```diff
@@ -0,0 +1,80 @@
+package utils;
+
+import data.Company;
+import data.Department;
+import data.Employee;
+import java.io.*;
+import java.util.ArrayList;
+
+public class FileOperations {
+    private final String fileName;
+    private Department nullDept = new Department("nullDepartment");
```

Comment:
to pasuje jako publiczne statyczne pole w klasie Department

Reply:
poprawione